rules/android_sdk_repository/helper.bzl

```diff
@@ -80,6 +80,42 @@ def _create_config_setting_rule():
         flag_values = {":allow_proguard": "false"},
     )
 
+def create_dummy_sdk_toolchain():
```

Comment:
this is required otherwise you get a toolchain resolution failure when it is looking for a android toolchain targeting the host. This likely shouldn't happen, but this is what bazel itself is doing so it's probably fine for now

Reply:
When will it happen? In normal situation, aren't users always build apps targeting an Android platform with specific API level? Is it for avoiding setting up the target platforms/API level in examples?